src/neptune/management/internal/api.py

```
@@ -563,6 +568,86 @@ def get_workspace_service_account_list(workspace: str, *, api_token: Optional[st
+    *,
+    username: Optional[str] = None,
+    email: Optional[str] = None,
+    workspace: str,
+    api_token: Optional[str] = None,
+    role: WorkspaceMemberRole = WorkspaceMemberRole.MEMBER,
+    add_to_all_projects: bool = False,
+) -> None:
+    """Creates invitation to Neptune workspace.
+
+    Args:
+        username: Username of the user to invite.
+        email: Email address of the user to invite.
+            Note: at least one of the above parameters is needed.
+            If both are filled email will be ignored.
+        workspace: Name of your Neptune workspace.
+        api_token: Account's API token.
+            If None, the value of the NEPTUNE_API_TOKEN environment variable is used.
+            Note: To keep your token secure, use the NEPTUNE_API_TOKEN environment variable rather than placing your
+            API token in plain text in your source code.
+        role: The workspace role that is to be granted to the invited user.
+        You can choose between the following values:
+        - Administrator: `WorkspaceMemberRole.ADMIN`
+        - Member: `WorkspaceMemberRole.MEMBER`
+        add_to_all_projects: Whether to add the user to all projects in the workspace.
+
+    Example:
+        >>> from neptune import management
+        >>> from management import WorkspaceMemberRole
+        >>> management.invite_to_workspace(
+        ...     username="user",
+        ...     workspace="ml-team",
+        ...     role=WorkspaceMemberRole.ADMIN,
+        ... )
+
+    You may also want to check the management API reference:
+    https://docs.neptune.ai/api/management
+    """
+    verify_type("workspace", workspace, str)
+    verify_type("role", role, WorkspaceMemberRole)
+    verify_type("add_to_all_projects", add_to_all_projects, bool)
+    verify_type("username", username, (str, type(None)))
+    verify_type("email", email, (str, type(None)))
+    verify_type("api_token", api_token, (str, type(None)))
+
+    if username:
```

We should raise an exception in case of setting both `username` and `email`.